doc/man/lttng-relayd.8.txt

```diff
@@ -220,6 +220,11 @@ Supported on Linux only.
 +
 Search for `tcp_keepalive_intvl` in man:tcp(7) for more information.
 
+`LTTNG_RELAYD_CWD`::
+    Working directory for the LTTng-relayd processes.
```

Comment:
Let's use:

    Working directory of the processes this relay daemon creates.

Reply:
done